src/bundle/Controller/ContentOnTheFlyController.php

```diff
@@ -70,6 +93,10 @@ public function createContentAction(
         ContentType $contentType,
         Location $parentLocation
     ) {
+        if ((new ContentTypeIsUser($this->userContentTypeIdentifier))->isSatisfiedBy($contentType)) {
+            return $this->createUserAction($request, $languageCode, $contentType, $parentLocation);
+        }
+
         $language = $this->languageService->loadLanguage($languageCode);
```

Comment:
We can move this line before it and then pass Language object to createUserAction method. Then it is not needed to load language in createUserAction method. I know that we are using language code, but I think we should start use object instead in new methods if it is possible.